<launch>
  <param name="port_name" value="/dev/ttyS19"/>
  <param name="baud_rate" value="9600"/>
  <node name="parse_serial_node" pkg="parse_serial" type="parse_serial_node"/>
</launch>